src/AppAny.Quartz.EntityFrameworkCore.Migrations.MySql/EntityTypeConfigurations/QuartzBlobTriggerEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AppAny.Quartz.EntityFrameworkCore.Migrations.Quartz;
+
+namespace AppAny.Quartz.EntityFrameworkCore.Migrations.MySql.EntityTypeConfigurations
```

Comment:
PostgreSQL project uses `AppAny.Quartz.EntityFrameworkCore.Migrations.PostgreSQL` namespace without EntityTypeConfigurations. All IEntityTypeConfigurations is a public api to allow extending each model and `.EntityTypeConfigurations` seems redundant